src/weights.jl

```diff
@@ -345,10 +414,10 @@ end
     wmedian(v, w)
 
 Compute the weighted median of an array `v` with weights `w`, given as either a
-vector or `WeightVec`.
+vector or `AbstractWeights`.
 """
-wmedian(v::RealVector, w::RealVector) = median(v, weights(w))
-wmedian{W<:Real}(v::RealVector, w::WeightVec{W}) = median(v, w)
+wmedian(v::RealVector, w::RealVector) = median(v, weights(w, false))
```

Comment:
If the `weights` API is changing to require a boolean parameter, the existing API should have a formal deprecation.